### PR TITLE
feat: add bell icon

### PR DESCRIPTION
**Pull request by dani-walnut**
## What

Add a Bell icon to Basics - it is used in Figma but missing in Wave 

### Media

​

## Why

​

### CODE REVIEW

**Review by martimalek — APPROVED** [review]

File: assets/icons/basic/bell.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<svg width="24" height="25" viewBox="0 0 24 25" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (arturmiglio) [inline assets/icons/basic/bell.svg:1]: The icons should all be 24x24...

File: src/icons/basic/BellIcon.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+            {...props}
+            width={sizePx}
+            height={sizePx}
+            viewBox="0 0 24 25"
+            fill="none"
+            xmlns="http://www.w3.org/2000/svg"
+        >
+            <path
+                d="M5 19.5a.967.967 0 01-.713-.288A.968.968 0 014 18.5c0-.283.096-.52.287-.712A.967.967 0 015 17.5h1v-7c0-1.383.417-2.612 1.25-3.688.833-1.075 1.917-1.779 3.25-2.112V4c0-.417.146-.77.438-1.063A1.447 1.447 0 0112 2.5c.417 0 .77.146 1.063.438.291.291.437.645.437 1.062v.7c1.333.333 2.417 1.037 3.25 2.112C17.583 7.888 18 9.117 18 10.5v7h1c.283 0 .52.096.712.288.192.191.288.429.288.712s-.096.52-.288.712A.968.968 0 0119 19.5H5zm7 3c-.55 0-1.02-.196-1.412-.587A1.926 1.926 0 0110 20.5h4c0 .55-.196 1.02-.588 1.413A1.926 1.926 0 0112 22.5zm-4-5h8v-7c0-1.1-.392-2.042-1.175-2.825C14.042 6.892 13.1 6.5 12 6.5s-2.042.392-2.825 1.175C8.392 8.458 8 9.4 8 10.5v7z"
+                fill="#1F191A"
```
Comment (arturmiglio) [inline src/icons/basic/BellIcon.tsx:22]: Not sure what happened here, but the fill should be 'currentColor'.

Maybe an issue with icon generator script?

**Review by arturmiglio — CHANGES_REQUESTED** [review]

File: src/icons/basic/BellIcon.tsx (hunk 2 above)
Comment (dani-walnut) [reply src/icons/basic/BellIcon.tsx:22]: @martimalek as you were updating the script recently?

Comment (martimalek) [reply src/icons/basic/BellIcon.tsx]: It was an issue with the source svg file, I've added a suggestion to make the change there directly

**Review by arturmiglio — APPROVED** [review]

**Pull request merged**